core.py

```
@@ -0,0 +1,431 @@
+                await self.writer.wait_closed()
+            except Exception:
+                pass
+        if self.on_disconnected:
+            self.on_disconnected()
+            
+    async def send_message(self, message: dict):
+        try:
+            data = json.dumps(message).encode('utf-8') + b'\n'
+            self.writer.write(data)
+            await self.writer.drain()
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)